src/RoadRunnerStore.php

```diff
@@ -16,14 +16,15 @@ final class RoadRunnerStore implements SharedLockStoreInterface
 {
     public function __construct(
         private readonly RrLockInterface $rrLock,
+        private float $initialTtl = 300.0,
```

Comment:
Note, that TTL units are microseconds (type - integer). RR treats all time coming to it as microseconds.

Reply:
This line simmilar to another realizations of [SharedLockStoreInterface](https://github.com/symfony/lock/blob/cde6dbd72d217024b1049d42a4519e713e2f18af/Store/RedisStore.php#L40)

I just pass this value to another class from `roadrunner-php/lock` and it already converts it to microseconds
https://github.com/roadrunner-php/lock/blob/6ad625ba1cb0ed67a55c04e4ed1cc6a0c0a3b78d/src/Lock.php#L46

Reply:
Do you mean `300.0` - seconds and it would be converted to microseconds - 3*10-8 (actually this value would be passed to the RR)?

Reply:
Yep
```
return (int) round($ttl * 1_000_000);
```